### DISCUSSION TITLE
Summer 2022 Workshop Recording

### DISCUSSION THREAD

**Discussion by thomas-allard** [idaholab/moose#21308] [Q&A Getting Started] [answered]
When can we expect the recording from this summer's training to be available? Will it be posted to this youtube channel: https://www.youtube.com/channel/UClljZCDkwVHSvBxK-fKMxMA ?

Thanks!

**Comment by loganharbour** [comment]
Sorry for the delay. The Teams recording is proving to be a little more difficult to cut up than expected. It will be posted to the same channel either tomorrow or at the beginning of next week (this Friday is our day off).

**Reply by thomas-allard** [reply]
Logan, that would be great, thanks!

**Reply by thomas-allard** [reply]
Any update? I've been hoping to watch the sections on the tensor mechanics and solid mechanics modules.

**Reply by GiudGiud** [reply]
It's pretty labor intensive to edit the video, sorry for the delay, we'll get to it

**Comment by thomas-allard** [comment]
Is there any update on this? 

I'm very interested in at least watching the tensor and solid mechanics portions of the workshop

**Reply by GiudGiud** [reply, marked as answer]
We've had issues editing the videos and dropped it. The hybrid setup was not good. We can have a look at those two parts specifically though there will be a delay

**Reply by thomas-allard** [reply]
Ok gotcha. Are there plans for another workshop sometime in 2023?

**Reply by GiudGiud** [reply]
There will be a workshop second week of June
We will announce it momentarily
